about.html

```diff
@@ -0,0 +1,92 @@
+                </div>
+            </div>
+            <ul class="nav-link">
+               <div class="menu-closebtn"><i class="fa-solid fa-xmark"></i></div>
+               <li class ="list"> <a href="index.html">Home</li></a>   
+               <li class ="list"> <a href="about.html">About</li></a>   
+               <li class ="list"><a href="#">Program</li></a>
+               <li class ="list"><a href="#">join</li></a>
+               <li class ="list"><a href="#">sponsor</li></a>
+               <li class ="list"><a href="#">News</li></a>
+               <li class ="list"><a href="#" class="liste">CC campaign</li></a>
+            </ul>
+        </nav>
+    </header>
+    <main>
+
+        <section class="main-top">
+            <div class="background">
+               <div class="message">
+                  <h1>international Fitness  <br> Submit conference <br> 2022</h1>
+                 <p class="fit">The must attend business festival for health, fitness, wellness, hospitality, spa and HR executives is back to become LIVE and IN-PERSON. Experience an immersive multi-day thought leadership conference, exhibition, start-up showcase and industry awards dinner, with multiple networking events, product launches, activation events, site visits and workouts across the week to showcase new trends, products, concepts, technologies, innovation, start-ups, franchises and emerging investment opportunities.</p>
+                  <p class="date">Please contact us per email for any further questions about international fitness  <br></p>
+                 <h4>thefitsummit.com <i class="fa-solid fa-angle-right"></i></h4>
+               </div>
+            </div>
+          </section>
+          <section class="sponsor-main">
+                <h2>international Fitness submit logo</h2>
+                <p>the logo of international fitness was decided through the logo competition 2021 from 10th august to 10th septmber.
+                    after online public voting and selection commitee judging process. the logo won the price and it is used for our submit through out the years
+                </p>
+
+                <img src="asset/fit-removebg-preview.png" alt="">
+          </section><hr>
+          <section class="logo">
+            <div class="top-list">
+            <h3>see the past fitness submit</h3>
+            <p>Take a look at the last two international fitness submit which took place at india </p>
+        </div>
+            <div id="start">
+                
+                    <div class="cont">
+                    <div class="one">    
+                         <h3>2017</h3>
+                        <p>internation submit in india</p>
+                     </div>
+                    </div>
+                <div class=" two">
+                    <h3>2018</h3>
+                    <p>internation submit in india</p>
+                </div>
+            </div>
+          </section>
+    
+    </main>
```

Comment:
- [ ] Pİctures are smaller in the original kindly change it.
- [ ] Font family is different. Kindly change it to all pages.
- [ ] `See The Past Fitness Submit title has a red horizontal line. Kindly add it
![image](https://user-images.githubusercontent.com/38635158/175399400-ace4c4af-4a5b-4a3a-96b8-8dff6e4181fb.png)

- [ ] ORIGINAL

![image](https://user-images.githubusercontent.com/38635158/175399451-58619dd2-1f4d-4a55-9ae6-f7ac653ea7c2.png)

### MOBİLE VERSİON
- [ ] Texts on the picture should be smaller. Kindly change it.

![image](https://user-images.githubusercontent.com/38635158/175400013-b436d109-44aa-4d30-82b3-e37e95016454.png)


### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/175400080-dbe59143-2e93-44cb-98cf-95e38f03dc87.png)